packages/bridge/src/vl/histogram/NumericalHistogramFilter.ts

```diff
@@ -172,8 +181,8 @@ export class NumericalHistogramFilter extends BaseHistogramFilter<[number, numbe
     if (evt.detail === null) {
       this._selection = null;
     } else {
-      const selection = (this._isTimeSeries ? evt.detail : evt.detail.selection) as [number, number];
-      this._selection = [Number(selection[0]), Number(selection[1])];
+      const selection = (this._isTimeSeries ? evt.detail : evt.detail.selection) as [number , number];
```

Comment:
I think Tslint should complain about this space but I dunno, it might not be ran on CI

```suggestion
      const selection = (this._isTimeSeries ? evt.detail : evt.detail.selection) as [number, number];
```